dev/com.ibm.ws.anno_fat/fat/src/com/ibm/ws/tests/anno/caching/extrascan/AbstractAnnotationScanInJarTest.java

```diff
@@ -0,0 +1,164 @@
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions;
+import com.ibm.websphere.simplicity.config.ConfigElementList;
+import com.ibm.websphere.simplicity.config.EnterpriseApplication;
+import com.ibm.websphere.simplicity.config.ServerConfiguration;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.EERepeatActions;
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+import junit.framework.Assert;
+import spring.test.init.jar.JarInit;
+import spring.test.init.manifest.ManifestInit;
+import spring.test.init.sharedlib.SharedLibInit;
+import spring.test.init.war.WebInit;
+
+@RunWith(FATRunner.class)
+public abstract class AbstractAnnotationScanInJarTest extends FATServletClient {
+
+	public static final String APP_NAME = "springTest";
+	public static final String SERVER_NAME = "springTest_server";
+
+	@Server(SERVER_NAME)
+	public static LibertyServer server;
+
+	@ClassRule
+	public static RepeatTests r = EERepeatActions.repeat(SERVER_NAME, true,
+			EERepeatActions.EE10,
+			EERepeatActions.EE11);
+
+	//Since there is only one test we can do the setup as part of the test to allow the use of abstraction
+	public void setUp() throws Exception {
+
+		WebArchive war = ShrinkWrap.create(WebArchive.class, APP_NAME + ".war")
+				.addPackages(true, WebInit.class.getPackage())
+				.addAsManifestResource(new StringAsset("Manifest-Version: 1.0" + System.lineSeparator() +
+						"Class-Path: manifestLib.jar" + System.lineSeparator()), "MANIFEST.MF") //The Class-Path will not be included without that trailing newline. See https://docs.oracle.com/javase/tutorial/deployment/jar/modman.html
+				.addAsWebInfResource(new StringAsset("logging.level.org.springframework.context.annotation=DEBUG"), "application.properties");
+
+		String userDir = System.getProperty("user.dir"); //ends with: com.ibm.ws.anno_fat/build/libs/autoFVT
+		String springDir = userDir + "/publish/shared/resources/spring/";
+
+		JavaArchive jar = ShrinkWrap.create(JavaArchive.class, APP_NAME + ".jar")
+				.addPackages(true, JarInit.class.getPackage());
+
+		JavaArchive sharedLib = ShrinkWrap.create(JavaArchive.class, "sharedLib.jar")
+				.addPackages(true, SharedLibInit.class.getPackage());
+
+		JavaArchive maifestJar = ShrinkWrap.create(JavaArchive.class, "manifestLib.jar")
+				.addPackage(ManifestInit.class.getPackage());
+
+		EnterpriseArchive ear = ShrinkWrap.create(EnterpriseArchive.class, APP_NAME + ".ear")
+				.addAsModule(war)
+				.addAsModule(maifestJar)
```

Comment:
It is a small nit, but I figure you want the variable name to be `manifestJar`

```suggestion
		JavaArchive manifestJar = ShrinkWrap.create(JavaArchive.class, "manifestLib.jar")
				.addPackage(ManifestInit.class.getPackage());

		EnterpriseArchive ear = ShrinkWrap.create(EnterpriseArchive.class, APP_NAME + ".ear")
				.addAsModule(war)
				.addAsModule(manifestJar)
```